contrib/question_converters/canvas/quiz2pl.py

```diff
@@ -158,7 +220,10 @@ def clean_question_text(text: str) -> str:
             or question["question_type"] == "essay_question"
         ):
             obj["gradingMethod"] = "Manual"
-        json.dump(obj, info, indent=4)
+        json.dump(obj, info, indent=2)
+
+    # Handle images.
+    question_text = handle_images(question_dir, question_text)
 
     with open(os.path.join(question_dir, "question.html"), "w") as template:
```

Comment:
Maybe a nit but instead of putting all of this under this a single with clause, the string should be assembled in memory and then written to the template file (i.e., move this with block to the end and only use a single write line to write the string to the file).

I think this might require a more comprehensive change, so if it's too much effort don't worry about it.

Reply:
Yeah, this is a somewhat invasive change that'll obscure the diff. I'd rather leave this for another PR.